From 16ecabd9d82b24661cb0b4d913bfd77c55be2e18 Mon Sep 17 00:00:00 2001
From: Alexander Shiyan <shc_work@mail.ru>
Date: Mon, 19 May 2014 13:21:17 +0800
Subject: [PATCH 002/125] ARM: i.MX: Select HAVE_IMX_SRC for i.MX5 globally

No reason to choose a symbol HAVE_IMX_SRC separately for each supported
i.MX5 CPU, this patch selects this symbol globally for i.MX5.

Signed-off-by: Alexander Shiyan <shc_work@mail.ru>
Signed-off-by: Shawn Guo <shawn.guo@freescale.com>
---
 arch/arm/mach-imx/Kconfig | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/arch/arm/mach-imx/Kconfig b/arch/arm/mach-imx/Kconfig
index 4b51857..a4da79c 100644
--- a/arch/arm/mach-imx/Kconfig
+++ b/arch/arm/mach-imx/Kconfig
@@ -123,11 +123,11 @@ config SOC_IMX5
 	bool
 	select ARCH_HAS_OPP
 	select ARCH_MXC_IOMUX_V3
+	select HAVE_IMX_SRC
 	select MXC_TZIC
 
 config	SOC_IMX51
 	bool
-	select HAVE_IMX_SRC
 	select PINCTRL_IMX51
 	select SOC_IMX5
 
@@ -702,7 +702,6 @@ comment "Device tree only"
 
 config	SOC_IMX50
 	bool "i.MX50 support"
-	select HAVE_IMX_SRC
 	select PINCTRL_IMX50
 	select SOC_IMX5
 
@@ -717,7 +716,6 @@ config MACH_IMX51_DT
 
 config	SOC_IMX53
 	bool "i.MX53 support"
-	select HAVE_IMX_SRC
 	select PINCTRL_IMX53
 	select SOC_IMX5
 
-- 
2.0.1

